### PR TITLE
ENG-124: fix preview panel flicker

### PR DESCRIPTION
**Pull request by AfeezGL**

### CODE REVIEW

File: apps/frontend/src/modules/surveys/components/studio/create/preview-panel/index.tsx [hunk 1]
```diff
@@ -42,7 +43,7 @@ export const Preview = ({ className, mode, viewPort, ...props }: Props) => {
                     ...survey,
                     theme: theme ? { ...theme, colorScheme: JSON.stringify(theme.colorScheme) } : {},
                     questions: parsedQuestions.map((q) => {
-                        if (q.id === question?.id) {
+                        if (q.orderNumber === question?.orderNumber) {
```
Comment (jdamilola) [inline apps/frontend/src/modules/surveys/components/studio/create/preview-panel/index.tsx:46]: @AfeezGL Why did you use `orderNumber` here instead of `id`? Remember, this will mutable when we implement reordering survey questions.

File: packages/web-sdk/src/core/root-frame.ts [hunk 2]
```diff
@@ -1,44 +1,50 @@
...
+        this.element.id = "integraflow-container";
+        this.containers = [];
+
+        onDOMReady(() => document.body.appendChild(this.element));
+    }
+
+    createContainer(name: string) {
+        const element = document.getElementById(`integraflow-container-${name}`);
+
+        if (element) {
```
Comment (jdamilola) [inline packages/web-sdk/src/core/root-frame.ts:25]: @AfeezGL If the element exists, don't we also need to remove it from the container list?

File: packages/web-sdk/src/index.ts [hunk 3]
```diff
@@ -28,11 +28,6 @@ export default class Integraflow {
         if (!this.client) {
             this.client = new Integraflow(config);
             this.initialized = true;
-        } else {
```
Comment (jdamilola) [inline packages/web-sdk/src/index.ts:31]: @AfeezGL Why did you remove this?

**Review by jdamilola — CHANGES_REQUESTED** [review]
Can you address my comments?

File: packages/web-sdk/src/index.ts (hunk 3 above)
Comment (AfeezGL) [reply packages/web-sdk/src/index.ts:31]: @jdamilola This else block was added to `Integraflow` class to support context updates in the preview panel, this block has now been moved to the internal `_Integraflow` class.

File: apps/frontend/src/modules/surveys/components/studio/create/preview-panel/index.tsx (hunk 1 above)
Comment (AfeezGL) [reply apps/frontend/src/modules/surveys/components/studio/create/preview-panel/index.tsx]: New questions were not showing after creation because the `id` is temporary until `createQuestionMutation` completes so there's a point where the `question.id` is different from the one from the same question in `parsedQuestions`. 
I used `orderNumber` to find the created question, I have now changed that to `reference`.

**Review by jdamilola — APPROVED** [review]
